celesta-sql/src/main/java/ru/curs/celesta/dbutils/CsqlBasicDataAccessor.java

```diff
@@ -77,15 +78,14 @@ public final void close() {
 
     protected abstract void closeInternal();
 
-    public abstract void clear() ;
+    public abstract void clear();
 
     /**
      * Объект метаданных (таблица, представление или последовательность), на основе которого создан
      * данный объект доступа.
      *
-     * @
-     *             в случае ошибки извлечения метаинформации (в норме не должна
-     *             происходить).
+     * @ в случае ошибки извлечения метаинформации (в норме не должна
```

Comment:
In eanglish, please. I understand, that this changes are related with formatting.  So there is not problem if you don't want to fix it.